.travis.yml

```diff
@@ -26,6 +25,4 @@ before_script:
 script:
   - bundle exec rake rubocop
   - bundle exec rake test:validate
-  - bundle exec rake check:matching_docs
```

Comment:
Don't we still need an equivalent of this since the actual apidoc is generated from code?

Reply:
If we want to ensure the docs are up to date, we can generate them each time the system starts up. We can do it with `unless File.exists` statement, so most of the time system startup time will not be impacted. Another option would be generating the doc as part of container creation.